lib/DropdownMenu/index.js

```diff
@@ -34,12 +35,14 @@ class DropdownMenu extends Component {
     direction: 'down',
     shouldDisplay: true,
     fullWidth: false,
+    buttonClassName: null,
```

Comment:
i think this might have to be changed to `''`, I swear I've seen it try to use 'null' as a class before (i could be remembering a really boring dream though)